src/routes/sdk/controllers/addSampleCrimes.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-param-reassign */
+import { Context, Next } from 'koa';
+import { ICrime } from '../../../models/Crime';
+import addCrime from '../services/addCrime';
+import makeSamples from '../services/makeSampleCrimes';
+
+const SAMPLE_COUNT = 1000;
+
+export default async (ctx: Context, next: Next): Promise<void> => {
+  const newCrimes: ICrime[] = makeSamples(SAMPLE_COUNT);
+  const { projectId } = ctx.params;
+  const userIp = ctx.request.ip;
+  newCrimes.map((newCrime) => {
+    newCrime.projectId = projectId;
+    newCrime.meta.ip = userIp;
+  });
+  try {
+    await Promise.all(
+      newCrimes.map(async (newCrime) => {
+        await addCrime(newCrime, projectId);
+      }),
+    );
+    ctx.response.status = 200;
+  } catch (e) {
+    ctx.throw(400, 'validation failed');
+  }
+  await next();
```

Comment:
저희 error handling 방법 정한대로 처리해주시면 좋을것 같습니다!
```suggestion
    ctx.throw(400);
  }
```

Reply:
1000개가 한꺼번에 들어가는건 아무래도 무리가 있을것같아보이네요 ... 흠
뭔가 별도의 로직이 필요할것도 같습니다. 
rdb에서 bulkinsert 같은 기능이 mongodb에서 insertMany가 있기는 한것같은데 저희는 들어갈때마다 issue 분리를 해줘야해서 머리가 아프네요 흠...

Reply:
수정 완료하였습니다!!